doxygen/dox/VOLConnGuide.dox

```diff
@@ -696,14 +706,17 @@ Retrieves an underlying object.
 <tr>
 <td>
 \code
-  obj  (IN): Object being unwrapped.
+  obj  (IN): The object to be unwrapped.
 \endcode
 </td>
 </tr>
 </table>
 
 \subsubsection subsubsecVOLRefWrapctx wrap: get_wrap_ctx
 Get a VOL connector's object wrapping context.
+
+The context should be returned in dynamically allocated memory under <em>*wrap_ctx</em>.
```

Comment:
@qkoziol When trying to help with documenting the VOL wrap class, I couldn't quite figure out what the wrap context object was necessary for. Do you happen to remember why a passthrough VOL needs to return a wrapping context object for the library to keep around?

Reply:
The wrapping context object is saved on the API Context, which is where `H5VL__wrap_obj` retrieves it from to provide it to the wrap object callback

Reply:
Wrapping objects is necessary for the library to return objects from user callbacks and other scenarios where the library has an object from a terminal VOL connector and needs to eventually pass back an ID to an application.  The library has to wrap the terminal object using pass-throughs 'wrap' callbacks, so that the resulting ID refers to an object that can later be unwrapped by the connector stack and reach the terminal connector again.

Reply:
Right, I was asking more about the wrapping context object in particular and why a connector needs to allocate one and pass it back to the library.

Reply:
The wrapping context is what the library gives back to a VOL connector to say "wrap this object for me".  Without the wrapping context, the pass-through connector wouldn't know which connector stack it was supposed to be wrapping an object for.